lib/sycamore/sycamore/llms/llms.py

```diff
@@ -10,11 +10,11 @@ def __init__(self, model_name, cache: Optional[Cache] = None):
         self._cache = cache
 
     @abstractmethod
-    def generate(self, *, prompt_kwargs: dict, llm_kwargs: Optional[dict] = None) -> Any:
+    def generate(self, *, prompt_kwargs: dict, llm_kwargs: Optional[dict] = None) -> str:
```

Comment:
I dunno how its implemented right now but some invocations expect a dict response, e.g. extract_schema. It might just be returning a json though

Reply:
Nvm I see you found summarize was relying on the dict return